apps/backend/src/modules/users/user.repository.ts

```diff
@@ -126,6 +145,7 @@ class UserRepository implements Repository<UserEntity> {
 					createdAt: user.createdAt,
 					email: user.email,
 					firstName: user.userDetails.firstName,
+					hasUnreadNotifications: Boolean(+user.unreadNotifcationsCount),
```

Comment:
Why do you need to convert user.unreadNotificationsCount to a number?
But if you need it is better not to use unary plus
```suggestion
					hasUnreadNotifications: Boolean(Number(user.unreadNotificationsCount)),
```

Reply:
When database returns data it returns it of type any and set it to the fields of the model which instance was created, and when I'm trying to cast it to the Boolean, even if there will be 0, it gives me true, so i need to cast it for sure

Reply:
Okay, then please use `Number(user.unreadNotificationsCount)`